condor/condor_meter

```diff
@@ -711,7 +710,19 @@ def classadToJUR(classad):
 
     setIfExists(r.Queue, classad, "JobUniverse", "Condor's JobUniverse field", setstr=True)
     setIfExists(r.NodeCount, classad, 'MaxHosts', "max")
-
+    ########################################################################################
+    ########################################################################################
+    # Code added to send to arbitrary Ads SOFTWARE-2714
+    ArbitraryJobAds = GratiaCore.Config.getConfigAttribute("ArbitraryJobAdAttributes")
+    DebugPrint(5, "ArbitraryJobAd: %s" % str(ArbitraryJobAds))
+    Arbirtray_adlist = ArbitraryJobAds.split(",")
```

Comment:
Won’t that allow it to split every single character apart? Perhaps something like this:

`Arbitrary_adlist = re.split(r'[,\s]+', ArbitraryJobAds)`